src/app/helpers/functions_helper.rb

```diff
@@ -7,4 +10,63 @@ def render_code(ruby)
     sanitized = sanitize(formatter.format(lexer.lex(ruby)), tags: %w[div span], attributes: %w[style])
     tag.pre sanitized
   end
+
+  def to_html(str)
+    options = { input: 'GFM', auto_id_stripping: true }
+    kramdown_doc = Kramdown::Document.new(str, options)
+    kramdown_doc.to_html_from_markdown
+  end
+end
+
+module Kramdown
+  module Converter
+    class HtmlFromMarkdown < Html
```

Comment:
[Q] can you tell me more about this class? is it defined in one of the packages and you're monkey patching it? tell me more about how did you reach to this solution here.

Reply:
In order to customize the HTML converter I had to subclass the HTML converter and override the tags I won't to customize (or disable)[[1]](https://github.com/gettalong/kramdown/blob/master/lib/kramdown/converter/html.rb#L18-L29). Kardown parser uses `#method_missing` to instantiate a converter class in the `Kramdown::Converter` module and use it for converting the document [[2]](https://kramdown.gettalong.org/rdoc/Kramdown/Document.html).